### PR TITLE
Completed Hashing-1. Please Review

### PR DESCRIPTION
**Pull request by owaiskazi19**

### CODE REVIEW

File: Isomorphic-String.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            return False
+        
+        for i in range(0, len(s)):
+            ss = s[i]
+            tt = t[i]
+
+            ss = 'first'+ss
+            tt = 'second'+tt
+
+            if ss not in res:
```
Comment (Himanshu-Agarwal93) [inline Isomorphic-String.py:26]: Looks like you made a mistake in copying the solution. Initialization of `res` is missing.

**Review by Himanshu-Agarwal93 — COMMENTED** [review]
Good work!

**Comment by Himanshu-Agarwal93** [comment]
Good work!

File: Isomorphic-String.py (hunk 1 above)
Comment (owaiskazi19) [reply Isomorphic-String.py:27]: Left res here but added it on LC

**Pull request closed without merging**